src/common/util/CMakeLists.txt

```diff
@@ -37,7 +37,9 @@ add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${PUBLIC_HEADERS})
 add_library(openvino::util ALIAS ${TARGET_NAME})
 set_target_properties(${TARGET_NAME} PROPERTIES EXPORT_NAME util)
 
-target_link_libraries(${TARGET_NAME} PRIVATE ${CMAKE_DL_LIBS} PUBLIC openvino::pugixml)
+# TODO: make lib linkage portable e.g.: openvino/src/bindings/python/thirdparty/pybind11/tests/CMakeLists.txt
+target_link_libraries(${TARGET_NAME} PRIVATE ${CMAKE_DL_LIBS} PUBLIC openvino::pugixml stdc++fs )
```

Comment:
does it required new C++ standard ?

Reply:
No, the feature will be C++11 compatible. Where possible I plan to use experimental part of std filesystem lib and in reference tests for custom implementation. I would like the interface to be as close to std as possible to smoothly adapt future fixups to the newer CPP standard.